Tools/TestWebKitAPI/wpe/mock-platform/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+set_property(DIRECTORY . PROPERTY FOLDER "mock-platform")
+
+add_library(WPEMockPlatform MODULE
+    WPEDisplayMock.cpp
+)
+
+set_target_properties(WPEMockPlatform PROPERTIES
+    C_VISIBILITY_PRESET hidden
+    POSITION_INDEPENDENT_CODE TRUE
+    VISIBILITY_INLINES_HIDDEN TRUE
+    INTERFACE_INCLUDE_DIRECTORIES "${CMAKE_CURRENT_SOURCE_DIR}"
```

Comment:
Any reason to not list this below in the `target_include_directories()` command? Using the `INTERFACE_*` properties defines settings to be used by _other_ things that link against `WPEMockPlatform`&mdash;and there is nothing else that links to it, so I think this line should not be needed.

Reply:
Yup, just removing it seems to work